packages/@aws-cdk/aws-codepipeline/test/pipeline.test.ts

```diff
@@ -513,19 +514,17 @@ describe('test with shared setup', () => {
     pipeline.stage('Build').addAction(new FakeBuildAction({ actionName: 'debug.com', input: sourceArtifact }));
 
     // THEN
-    expect(stack).toHaveResourceLike('AWS::CodePipeline::Pipeline', {
-      Stages: arrayWith({
+    Template.fromStack(stack).hasResourceProperties('AWS::CodePipeline::Pipeline', {
+      Stages: Match.arrayWith([{
         Name: 'Build',
         Actions: [
-          objectLike({ Name: 'Gcc' }),
-          objectLike({ Name: 'debug.com' }),
+          Match.objectLike({ Name: 'Gcc' }),
+          Match.objectLike({ Name: 'debug.com' }),
         ],
-      }),
+      }]),
     });
   });
 });
-
-
```

Comment:
You went too far! One of these empty lines should be preserved. 🙂 (There should always be an empty line before `interface ...`).

```suggestion

```